src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -752,43 +746,24 @@ public static uint HeuristicDivide(ref BigInteger dividend, ref BigInteger divis
 
             public static uint LeadingZeroCount(uint value)
```

Comment:
This wrapper method is not necessary. The callers can call `BitOps.LeadingZeroCount` directly.

Reply:
Fixed